### ISSUE TITLE
Extended History cannot read my history

### ISSUE THREAD

**Issue by valmik** [chrome-enhanced-history/enhanced-history#9]
Seems like there's some data access problem. The only history visible are the times I viewed pdfs in browser. I disabled the extension, and regular history works fine, so it's definitely an issue with the way extended history interfaces with chrome.

**Comment by kkarakk** [comment]
yup it's broken, can't search past a couple of days anymore...normal history works fine
